src/Liquidator.sol

```diff
@@ -209,8 +211,12 @@ contract Liquidator is Ownable {
         AuctionInformation memory auctionInformation_ = auctionInformation[vault];
         require(auctionInformation_.inAuction, "LQ_BV: Not for sale");
 
-        uint256 priceOfVault =
-            _calcPriceOfVault(block.timestamp - auctionInformation_.startTime, auctionInformation_.openDebt);
+        uint256 timePassed;
+        unchecked {
```

Comment:
Isn't a checked subtraction cheaper than an unchecked subtraction + storage in memory?

Reply:
74 gas cheaper to do it unchecked vs `_calcPriceOfVault(block.timestamp - auctionInformation_.startTime, auctionInformation_.openDebt);`